src/WebJobs.Script.WebHost/Middleware/ExceptionMiddleware.Log.cs

```diff
@@ -11,14 +11,19 @@ internal partial class ExceptionMiddleware
         private static class Logger
         {
             private static readonly Action<ILogger, Exception> _responseStarted =
-               LoggerMessage.Define(LogLevel.Debug, new EventId(1, nameof(ResponseStarted)), "The response has already started, the status code will not be modified.");
+                LoggerMessage.Define(LogLevel.Debug, new EventId(1, nameof(ResponseStarted)), "The response has already started, the status code will not be modified.");
 
             private static readonly Action<ILogger, Exception> _unhandledHostError =
                 LoggerMessage.Define(LogLevel.Error, new EventId(2, nameof(UnhandledHostError)), "An unhandled host error has occurred.");
 
+            private static readonly Action<ILogger, Exception> _invocationCanceled =
+                LoggerMessage.Define(LogLevel.Warning, new EventId(3, nameof(InvocationCanceled)), "An invocation request was canceled before it was sent to the worker.");
```

Comment:
@fabiocav we already log something about this in GrpcWorkerChannel as a warning, do we need to log this again? And should we log one of them as info or debug?

Reply:
If we already have a log that covers this, as long as it is sufficient, no need for this additional one.